kubetest/util.go

```diff
@@ -216,7 +217,14 @@ func getChannelGKEVersion(project, zone, region, gkeChannel string) (string, err
 
 	for _, channel := range c.Channels {
 		if strings.EqualFold(channel.Channel, gkeChannel) {
-			return "v" + channel.DefaultVersion, nil
+			if len(channel.ValidVersions) == 0 {
+				return "", fmt.Errorf("channel doest not have valid versions")
+			}
+			if strings.EqualFold(extractionMethod, "latest") {
+				return "v" + channel.ValidVersions[0], nil
```

Comment:
It seems to assume that latest is the first one so the versions.
Do we have any guarantee about the order of ValidVersions in ReleaseChannelConfig?

https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1/ServerConfig#releasechannelconfig

Reply:
Actually, we do not have guarantees about it. I will add some simple sort logic for that.